### ISSUE TITLE
/var/log/turn_*.log are created

### ISSUE THREAD

**Issue by 532910** [coturn/coturn#602]
This is sync of https://bugs.debian.org/966671

with `/etc/turnserver.conf`:
```
no-tlsv1
no-tlsv1_1
listening-ip=<LOCAL_IP>
external-ip=<EXT_IP>
realm=domain.tld
denied-peer-ip=10.0.0.0-10.255.255.255
denied-peer-ip=192.168.0.0-192.168.255.255
denied-peer-ip=172.16.0.0-172.31.255.255
allowed-peer-ip=10.8.0.0-10.8.255.255
use-auth-secret
static-auth-secret=<SECRET>
user-quota=16
syslog
mobility
cert=/etc/ssl/localcerts/domain.tld-cert.pem
pkey=/etc/ssl/localcerts/domain.tld-key.pem
cipher-list="CHACHA20:AESGCM:!AES128"
dh2066
no-cli
cli-password=pass
```

it creates a lot of turn_*.log files

**Comment by misi** [comment]
Hi @532910  With which OS which coturn version do you experience this issue?

**Comment by 532910** [comment]
debian stable, 4.5.1.1-1.1+deb10u1

**Comment by misi** [comment]
@532910  
The problem is that we read options before we set the log destination until sequential read of options not reach the syslog it logs to file. As Quick fix / workaround put in the config file first line the syslog option. 

It has been fixed in the latest master in commit  5b13fdd 
Thanks for your patience..
